distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+==== Directory Permissions
+
+Restrict access to sensitive files by ensuring that the only users with access privileges are administrators.
+
+===== Directory Permissions on Windows
+
+. Right-click on the file or directory noted below then select *Full Control -> Administrators -> System*.
+. Click *Properties -> Security -> Advanced* and select `Creator Owner` for `${branding}_HOME` (e.g., `C:\ddf`).
+. Restrict access to sensitive files by ensuring that only *System* and *Administrators* have Full Control to the below files by right-clicking on the file or directory below then selecting *Properties -> Security -> Advanced*
+. Delete any other groups or users listed with access to `${branding}_HOME/etc` and `${branding}_HOME/deploy`.
+
+===== Directory Permissions on *NIX
+
+Protect the ${branding} from unauthorized access.
+
+. As root, change the owner and group of critical ${branding} directories to the `NON_ROOT_USER`.
++
+[WARNING]
+====
+A `NON_ROOT_USER` (e.g., ${branding-lowercase}) is recommended for installation.
+====
++
+[source, bash, linenums]
+----
+chown -R NON_ROOT_USER ${branding}_HOME ${branding}_HOME/etc ${branding}_HOME/data
+chgrp -R NON_ROOT_USER ${branding}_HOME/etc ${branding}_HOME/data
```

Comment:
We need to use a group name here instead of `NON_ROOT_USER`. We already mention `"group"` on line 40 so should we do something similar with the group like we did for the user?